framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -3489,8 +3442,41 @@ public TreePath getPathToCompilationUnit() {
     return TreePath.getPath(currentRoot, currentRoot);
   }
 
+  /**
+   * Index of this checker {@link #getSubcheckers()} or the size of {@link #getSubcheckers()} if
+   * this is the parent checker. Do not use this field directly. Call {@link #getSubCheckerIndex()}
+   * instead.
+   */
+  private int subcheckerIndex = -1;
+
+  /**
+   * Index of this checker {@link #getSubcheckers()} or the size of {@link #getSubcheckers()} if
```

Comment:
Should this be "in `getSubcheckers()`"?  Is it the index in the getSubcheckers of its immediate parent, or the ultimate parent?